src/release/main.d

```diff
@@ -859,15 +860,32 @@ SemVerBranch[] getBranches ( ref HTTPConnection con, ref Repository repo,
                v.minor.isNull;
     }
 
+    bool isSemVerBranch ( GitRef git_ref )
+    {
+        try
+        {
+            auto _ = SemVerBranch(git_ref.name);
+            return true;
+        }
+        catch (Exception)
+            return false;
+    }
+
     auto branches = repo.branches()
-                      .map!(a=>SemVerBranch(a.name))
-                      .filter!(a=>!thisVersion(a) &&
-                                  (newerVersion(a) ||
-                                  (same_major && ourMajorRelease(a))))
-                      .array();
+        .filter!isSemVerBranch
+        .map!(a=>SemVerBranch(a.name))
+        .filter!(a=>!thisVersion(a) &&
+            (newerVersion(a) ||
+            (same_major && ourMajorRelease(a))))
+        .array();
 
     branches.sort();
 
+    auto ignored_branches = repo.branches()
+        .filter!(name => !isSemVerBranch(name))
+        .map!(a => a.name);
+    writefln("List of ingored non-semver branches: %s", ignored_branches);
```

Comment:
Is this really useful information?

Reply:
Maybe for debugging ?

Reply:
But then it should be only enabled on request, having debugging info each time you run the tool would be a bit annoying.

Reply:
I thought it is important to make user acknowledge there are indeed some non-semver branches. For example, one could make a typo in branch name so that it becomes ignored and wonder why neptune doesn't work after.